.idea/jsDialects.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JSDialectsMappings">
```

Comment:
This will disallow ES6 in static code. Yay

Reply:
Yay indeed! :)